tests/test_deploy_qml.cpp

```diff
@@ -0,0 +1,111 @@
+// library includes
+#include <boost/filesystem.hpp>
+#include <gtest/gtest.h>
+
+// local includes
+#include "../src/qml.h"
+#include "../src/util.h"
+
+namespace bf = boost::filesystem;
+
+namespace LINUXDEPLOY_PLUGGIN_QT_TESTS {
```

Comment:
We wouldn't really need a namespace here I guess.

Please use a lower-case namespace. I suggest `linuxdeploy::plugin::qt::tests`.